### PR TITLE
Add npm stats to readme

### PR DESCRIPTION
**Pull request by kuldeepkeshwar**
## Description
Add npm stats to readme

## Motivation and Context
closes [123](https://github.com/siddharthkp/bundlesize/issues/123) 

## Types of changes

- Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by palashmon** [comment]
Could you please update the badge to show number of downloads/month? Or just add the badge in the original issue.

Thanks!

**Comment by kuldeepkeshwar** [comment]
We can use one of the below, let's agree on one badge 

[NPM](https://nodei.co/npm/bundlesize/)

Or

[npm](https://www.npmjs.com/package/bundlesize)

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
 [![NPM Version](https://img.shields.io/npm/v/bundlesize.svg)](https://npmjs.org/package/bundlesize)
 [![js-standard-style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg)](http://standardjs.com)
 
-[![NPM](https://nodei.co/npm/bundlesize.png?downloads=true&downloadRank=true&stars=true)](https://nodei.co/npm/bundlesize/)
+[![NPM](https://nodei.co/npm/bundlesize.png?months=1&downloads=true&downloadRank=true&stars=true)](https://nodei.co/npm/bundlesize/)
```
Comment (palashmon) [inline README.md]: Not showing anything related to monthly downloads count in the image yet. Its URL is same as here:
```
https://nodei.co/npm/bundlesize.png?months=1&downloads=true&downloadRank=true&stars=true
```
NPM Stats

Maybe it is still showing the previously cached data?

Comment (kuldeepkeshwar) [reply README.md]: yes seems like it's cached

File: README.md [hunk 2]
```diff
@@ -11,6 +11,7 @@
 [![NPM Version](https://img.shields.io/npm/v/bundlesize.svg)](https://npmjs.org/package/bundlesize)
 [![js-standard-style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg)](http://standardjs.com)
 
+[![NPM](https://nodei.co/npm/bundlesize.png?months=1&downloads=true&downloadRank=true&stars=true)](https://nodei.co/npm/bundlesize/)
```
Comment (reznord) [inline README.md]: This looks so bad in the README: Instead I'd go for something like:

[NPM Downloads](https://www.npmjs.com/package/bundlesize)

```
[![NPM Downloads](https://img.shields.io/npm/dm/bundlesize.svg?style=flat)](https://www.npmjs.com/package/bundlesize)
```

Comment (reznord) [reply README.md]: ^ the above is the badge for `bundlesize` repo

Comment (palashmon) [reply README.md]: I like this _shields.io_ badge too :heart_eyes:

Comment (reznord) [reply README.md]: Please change it to that, so I can merge this before next release (which @SaraVieira has to do)

Comment (SaraVieira) [reply README.md]: I agree with the above, that one is just too clunky

**Comment by reznord** [comment]
Looks much better, thanks! 👍

**Comment by palashmon** [comment]
@reznord I think it would be nice if we could see full download statistics for the package from the start. We just need to change the badge url from `https://www.npmjs.com/package/bundlesize` to `http://npm-stat.com/charts.html?package=bundlesize&from=2017-06-20` for that.

What do you say?

**Comment by reznord** [comment]
I wouldn't do that since it would be more like showing off the package.

Showing monthly downloads is just good enough!

**Comment by siddharthkp** [comment]
Oh I totally missed this, this is super cool!

**Pull request merged**